.travis.yml

```diff
@@ -9,15 +9,51 @@ branches:
 
 env:
     matrix:
-        - SYSTEM_PACKAGES=false
-        - SYSTEM_PACKAGES=true
+        - SYSTEM_PACKAGES=false QT=qt5.2
+        - SYSTEM_PACKAGES=true  QT=qt5.2
+        - SYSTEM_PACKAGES=false QT=qt5.4
+        - SYSTEM_PACKAGES=true  QT=qt5.4
```

Comment:
Let's use 3 builds instead of 4; I think there is no need to test SYSTEM_PACKAGES=false with two different Qt versions. It should be fine to test it only with qt5.2 or qt5.4.